libflux/src/flux/semantic/flatbuffers/tests.rs

```diff
@@ -0,0 +1,807 @@
+extern crate flatbuffers;
```

Comment:
I don't think you want to merge this yet, do you?

Reply:
Oops. Nope. Let me remove that.